index.html

```diff
@@ -37,7 +37,7 @@ <h1 id="title">Hello</h1>
     <textarea maxlength="500" cols="" rows="">
 
     </textarea>
-    <button type="submit" id="submit"></button>
+    <button type="submit" id="submit"> Submit</button>
    
   
         </select>
```

Comment:
Hi Please just as you have rightly done by sending each element of the form to a new line 👍 , Kinly sends the Button too to a new line. That is below the textarea